.gitmodules

```diff
@@ -26,7 +26,7 @@
 	path = src/doc/rust-by-example
 	url = https://github.com/rust-lang/rust-by-example.git
 [submodule "src/stdarch"]
```

Comment:
```suggestion
[submodule "library/stdarch"]
```

Reply:
Ugh, I made this change 3 times. Why didn't git pay attention?